Add security threat submission beta api docs (#16242)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/security-urlthreatsubmission-list.md

---
title: "List urlThreatSubmissions"
description: "Get a list of the urlThreatSubmission objects and their properties."
author: "caigen"
ms.localizationpriority: medium
ms.prod: "security"
doc_type: apiPageType
---

# List urlThreatSubmissions
Namespace: microsoft.graph.security

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

Get a list of the [urlThreatSubmission](../resources/security-urlthreatsubmission.md) objects and their properties.

## Permissions

> One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

Reviewer: ```suggestion
One of the following permissions is required to call this API. To learn more permissions, including how to choose one, see [Permissions](/graph/permissions-reference).
```

|Permission type|Permissions (from least to most privileged)|
|:---|:---|
|Delegated (work or school account)|ThreatSubmission.Read,ThreatSubmission.ReadWrite,ThreatSubmission.Read.All,ThreatSubmission.ReadWrite.All|
|Delegated (personal Microsoft account)|Not supported|
|Application|ThreatSubmission.Read.All,ThreatSubmission.ReadWrite.All|

## HTTP request

<!-- {
  "blockType": "ignored"
}
-->
``` http
GET /security/threatSubmission/urlThreats
```

## Optional query parameters

> This method supports some of the OData query parameters to help customize the response. For general information, see [OData query parameters](/graph/query-parameters).
> | Pattern                | Supported | Syntax                                 | Notes |
> | ---------------------- | :-------: | -------------------------------------- | ----- |
> | Server-side pagination |     ✓     | `@odata.nextLink`                      |       |
> | Filter                 |     ✓     | `/urlThreats?$filter=url eq 'value'` |       |
> | Filter                 |     ✓     | `/urlThreats?$filter=id eq 'value'` |       |
> | Filter                 |     ✓     | `/urlThreats?$filter=createdDateTime ge 'value' and createdDateTime lt 'value'` |       |
> | Filter                 |     ✓     | `/urlThreats?$filter=category eq 'value'` |       |
> | Filter                 |     ✓     | `/urlThreats?$filter=createdBy eq 'value'` |       |
> | Filter                 |     ✓     | `/urlThreats?$filter=status eq 'value'` |       |
> | Filter                 |     ✓     | `/urlThreats?$filter=source eq 'value'` |       |

Author: ```suggestion
This method supports `$filter`, `$top`, `$skipToken` and `$count` to help customize the response. For general information, see [OData query parameters](/graph/query-parameters).
```

## Request headers
|Name|Description|
|:---|:---|
|Authorization|Bearer {token}. Required.|

## Request body

> Do not supply a request body for this method.

Reviewer: ```suggestion
A request body is not required for this method. 
```

Author: It is **auto generated.** So let us keep it so it is aligned with other docs.
